index.html

```diff
@@ -28,9 +28,9 @@
 
     <!-- Sidebar -->
     <nav id="sidebar">
+      <a href="#about"><img src="picrew-img.png" /></a>
```

Comment:
If you do this, won't this not work with the bootstrap nav-link scrolling? Try this suggestion.
```suggestion
      <a href="#about" class="nav-link"><img src="picrew-img.png" /></a>
```

Reply:
Good catch, I think it will work with the on click function in `script.js`. I will give that a try.

Reply:
Tested it, the scrolling is back but there's now a border around the image because of the `.nav-link` styles. Let me find a better solution.